src/main/java/teammates/common/util/Config.java

```diff
@@ -95,7 +95,11 @@ private String getGcsBucketname() {
     }
 
     private String getAppUrl() {
-        return props.getProperty("app.url");
+        return Url.trimTrailingSlash(props.getProperty("app.url"));
+    }
+    
+    public static AppUrl getAppUrl(String relativeUrl) {
```

Comment:
I'm aware of that, and I believe that's a right thing to do? The alternative is throwing `MalformedURLException` in every method where a `Url` is constructed, and this will be a very widespread change.